packages/eslint-config-react/rules/import.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  overrides: [
+    /**
+     * This override enables default exports for files that require them.
+     * @link https://github.com/reside-eng/guidelines/blob/main/rfcs/shared-patterns/typescript/001-consistent-import-export.md
+     */
+    {
+      files: [
+        // Storybook stories
+        '**/*.stories.{jsx,tsx}',
+
+        // Next.js pages and API routes
+        'src/pages/**/*.{jsx,tsx}',
+        'pages/**/*.{jsx,tsx}',
+
+        // Next.js config
+        'next.config.{js,mjs}',
```

Comment:
Do we need to also include `jest.config.js` and `jest.setup.js` files?

Reply:
Hmm. I would think so, but when I ran eslint just now with the rules added locally, I didn't get any errors for the jest files 🤔 .

Reply:
I'll add them to be safe.

Reply:
Should that live in eslint-plugin-base, since we may use Jest for non-frontend projects?

Reply:
Yes, if it's needed, the base would make the most sense.